scripts/compute_taxonomy_breakdown.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2016--, Evguenia Kopylova.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+# Compute number of unique taxonomies and their counts
+
+import sys
+
+
+def get_genome_paths(scores_average_fp, scores_repophlan_fp):
+    """Return taxonomy strings and their counts.
+    """
+    genomes = []
+    # Get genome IDs
+    with open(scores_average_fp) as scores_average_f:
```

Comment:
there is a lot of parsing of tab delimited text in this PR. would it make sense to use pandas?